ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -33,10 +33,20 @@
 
 #include <string>
 
+/*
+ * In Azure: Buckets are synonymous with Containers, and Objects are synonymous with Blobs.
+ * This class represents an active connection to the Azure endpoint and allows for interaction with
+ * the Azure client. The azure_connection allows for the following functionality:
+ * Listing the container contents filtered by a given prefix and output all or output single, Puts a
+ * blob to the cloud, gets the blob from the cloud, and deletes a blob from the cloud. It also can check
+ * for the existance for a unique container or blob. Each azure_connection is associated with a unique
+ * azure_client with it's own unique container.
```

Comment:
Couple of grammar nitpicks, sorry for being a pain:

* ", Puts a" -> ", puts a"
* "gets the blob" -> "gets a blob"
* "for the existance for" -> "for the existence of" (note two fixes here)
* "with it's own" -> "with its own"

Reply:
hahahaaa no im really thankful for it :) changes made